src/macports1.0/reclaim.tcl

```diff
@@ -72,7 +72,7 @@ namespace eval reclaim {
             write_last_run_file disabled
             return
         }
-
+        
```

Comment:
Bogus whitespace was probably not intended here?

Reply:
yes, I'll make a lint commit

Reply:
@mojca  I'll make a lint commit for that :D

Reply:
@mojca  Changing whitespace doesn't show as a valid change on git. How do I fix this?

Reply:
That sounds unusual. What does `git status` say? It could be that you either forgot to do `git add`, or that you added the file, but then forgot the `--cached` flag in `git diff --cached` when you asked for changes. Or maybe you forgot to save the file, or the editor is playing with you (I have the editor configured in such a way that it always displays invisible characters, so that I can spot the difference between tabs vs. spaces, trailing whitespace etc.).